source/wp-content/themes/wporg-developer-2023/patterns/reference-content.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * Title: Reference Content
+ * Slug: wporg-developer-2023/reference-content
+ * Inserter: no
+ */
+
+?>
+
+<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"top":"var:preset|spacing|60","bottom":"var:preset|spacing|60","left":"var:preset|spacing|edge-space","right":"var:preset|spacing|edge-space"}}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group alignfull" style="padding-top:var(--wp--preset--spacing--60);padding-right:var(--wp--preset--spacing--edge-space);padding-bottom:var(--wp--preset--spacing--60);padding-left:var(--wp--preset--spacing--edge-space)">
+
+	<!-- wp:columns {"align":"wide","style":{"spacing":{"blockGap":{"left":"var:preset|spacing|20"}}}} -->
+	<div class="wp-block-columns alignwide">
+		
+		<!-- wp:column {"style":{"spacing":{"blockGap":"0"}}} -->
+		<div class="wp-block-column">
+
+			<!-- wp:wporg/reference-new-updated /-->
+
+		</div>
+		<!-- /wp:column -->
+	
+		<!-- wp:column {"style":{"spacing":{"blockGap":"0"}}} -->
+		<div class="wp-block-column">
+			
+			<!-- wp:group {"layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"space-between"}} -->
+			<div class="wp-block-group">
+				
+				<!-- wp:heading {"style":{"typography":{"fontStyle":"normal","fontWeight":"700"}},"fontSize":"normal","fontFamily":"inter"} -->
+				<h2 class="wp-block-heading has-inter-font-family has-normal-font-size" style="font-style:normal;font-weight:700">API reference</h2>
+				<!-- /wp:heading -->
+
+				<!-- wp:paragraph -->
+				<p><a href="https://developer.wordpress.org/apis/">View all</a></p>
+				<!-- /wp:paragraph -->
+			
+			</div>
+			<!-- /wp:group -->
+
+			<!-- wp:group {"style":{"spacing":{"padding":{"top":"var:preset|spacing|20","bottom":"var:preset|spacing|20","left":"var:preset|spacing|20","right":"var:preset|spacing|20"},"margin":{"top":"var:preset|spacing|20"}},"border":{"width":"1px"}},"borderColor":"light-grey-1","layout":{"type":"constrained"}} -->
+			<div class="wp-block-group has-border-color has-light-grey-1-border-color" style="border-width:1px;margin-top:var(--wp--preset--spacing--20);padding-top:var(--wp--preset--spacing--20);padding-right:var(--wp--preset--spacing--20);padding-bottom:var(--wp--preset--spacing--20);padding-left:var(--wp--preset--spacing--20)">
+			
+				<!-- wp:navigation {"ref":144716,"overlayMenu":"never","className":"wporg-reference-list","layout":{"type":"flex","orientation":"vertical"},"style":{"spacing":{"blockGap":"0"}},"fontSize":"small"} /-->
```

Comment:
Currently referencing this staging menu https://developer.wordpress.org/redesign-test/wp-admin/site-editor.php?postId=144716&postType=wp_navigation